constellations/gemini-admin/src/session/sessionEdit.js

```diff
@@ -0,0 +1,275 @@
+        }
+        if (modalContent) {
+            modalDiv = (
+                <Modal
+                    content={modalContent}
+                    show={showModal}
+                    onDismiss={this.onDismissModal}
+                />
+            );
+        }
+
+        return (
+            <div id="view-session-edit">
+                {modalDiv}
+                <h2>Edit Session</h2>
+
+                <div className="item">
+                    <h4>Choose a Day</h4>
+                    <SingleDatePicker
+                        id="date-picker"
+                        date={this.state.startsAt}
+                        onDateChange={(date) => this.onDateChange(date)}
+                        focused={this.state.dateFocused}
+                        onFocusChange={({ focused }) =>
+                            this.setState({
+                                dateFocused: focused,
+                            })
+                        }
+                        showDefaultInputIcon
+                    />
+                </div>
+
+                <div id="edit-times">
+                    <div className="item">
+                        <h4>Start Time</h4>
+                        <TimePicker
+                            timezone="America/New_York"
+                            time={this.state.startsAt.format("HH:mm")}
+                            theme="classic"
+                            onTimeChange={(options) =>
+                                this.onTimeChange("start", options)
+                            }
+                            timeMode="12"
+                            timeConfig={{
+                                from: "8:00 AM",
+                                to: "9:00 PM",
+                                step: 15,
+                                unit: "minutes",
+                            }}
+                        />
+                    </div>
+
+                    <div className="item">
+                        <h4>End Time</h4>
+                        <TimePicker
+                            timezone="America/New_York"
+                            time={this.state.endsAt.format("HH:mm")}
+                            theme="classic"
+                            onTimeChange={(options) =>
+                                this.onTimeChange("end", options)
+                            }
+                            timeMode="12"
+                            timeConfig={{
+                                from: "8:00 AM",
+                                to: "11:00 PM",
+                                step: 15,
+                                unit: "minutes",
+                            }}
+                        />
+                    </div>
+                </div>
+
+                <div id="cancel-toggle" className="item">
+                    <h4>Canceled</h4>
+                    <input
+                        type="checkbox"
+                        value={this.state.canceled}
```

Comment:
I think this is not the correct way to set a checkbox as checked or not.
You'll need something like this: https://stackoverflow.com/questions/39120007/setting-a-checkbox-check-property-in-react

You have to update the "checked" attribute of <input> in order to have it checked by default